### PR TITLE
virttest.guest_agent: Add function of guest-exec and guest-exec-status

### PR DESCRIPTION
**Pull request by xiagao**
ID: 1564829
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by lijinlijin** [comment]
Acked-by: Li Jin lijin@redhat.com

File: virttest/guest_agent.py [hunk 1]
```diff
@@ -822,5 +825,36 @@ def guest_file_seek(self, handle, offset, whence):
...
         :return: a dict with position and eof.
         """
-        return self._guest_file_operation("seek", handle=handle, offset=offset,
-                                          whence=whence)
+        cmd = "guest-file-seek"
+        return self._cmd_args_update(cmd, handle=handle, offset=offset,
+                                     whence=whence)
+
+    def guest_exec(self, path, arg=None, env=None, input_data=None,
+                   capture_output=None):
```
Comment (hereischen) [inline virttest/guest_agent.py]: `capture_output=False`

Comment (xiagao) [reply virttest/guest_agent.py]: No, if capture_output =None, guest-exec cmd will use default value(false)

Comment (xiagao) [reply virttest/guest_agent.py]: if capture_output =None, `_cmd_args_update() ` function will handle it to delete this key, then guest-exec cmd will use the default value.
such as:  {"execute": "guest-exec", "arguments": {"path": "echo", "arg": ["This", "is", "a", "test."]}}
this qga cmd will adopt the default `capture_output` value.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**